Remora.Discord.Commands/Conditions/RequireContextCondition.cs

```diff
@@ -68,21 +69,42 @@ public async ValueTask<Result> CheckAsync(RequireContextAttribute attribute, Can
 
             var channel = getChannel.Entity;
 
-            return attribute.Context switch
+            if (attribute.ChannelTypes != null)
```

Comment:
Use `is null` and `is not null` where appropriate, depending on which property you keep.